### DISCUSSION TITLE
Improve Translation Strings management

### DISCUSSION THREAD

**Discussion by bcartier** [directus/directus#15503] [Feature Requests]
Currently, under /admin/settings/translation-strings , all strings are listed in one table, and it only shows the key and one language value . 

1. It would be more performant and a better user experience to have the standard search and pagination used in other tables throughout the system here. 

2. It would be nice to be able to add additional columns to show the translated value for another language at a glance, rather than having to open the detail drawer for every item.

3. It would be great to be able to filter the list to find keys with missing values for a given language.

**Comment by rijkvanzanten** [comment]
This is being worked on and will go live soon! https://github.com/directus/directus/pull/18097
